Space-Runners/ablo-demo-fe
# Cu 866arfafx/build bottom toolbar/mihovil kovaevi

### Description (what's changed?)

Redesigned the bottom toolbar

### Testing instructions

1. Notice the new icons
2. Try to make a new AI Image and notice new color palette images
3. Try font to image or text to image. There'll be a coming soon screen
